lesson01/assignment/index.html

```diff
@@ -12,13 +13,55 @@
 <body>
   <main>
     <h1>Using ES6+ Methods in DOM Interactions</h1>
+    <h2>Example</h2>
     <form id="example-form">
       <label>Enter your favorite number:
         <input type="text" class="input">
       </label>
       <input type="submit" value="See Demo!">
       <p>The total of my array elements summed with your favorite number is: <span class="output"></span></p>
     </form>
+
+    <h2>1. Create a New Array Using Map by multiplying each element by the input value</h2>
+    <p>Array: [1, 2, 3, 4, 5, 6]</p>
```

Comment:
Excellent labeling for user experience. I'd move this `h2` and `p` inside the `form`. Also consider populating the `p` from the javascript so it's the same exact array that's being manipulated in the script.